runtime/common/src/lib.rs

```diff
@@ -83,15 +83,79 @@ macro_rules! decl_common_types {
         use scale_info::TypeInfo;
         use sp_core::storage::ChildInfo;
         use sp_runtime::{
-            generic, DispatchError, DispatchResult, RuntimeDebug, SaturatedConversion,
+            generic, BoundedBTreeMap, DispatchError, DispatchResult, RuntimeDebug,
+            SaturatedConversion,
         };
         use zeitgeist_primitives::traits::{DeployPoolApi, DistributeFees, MarketCommonsPalletApi};
 
         pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 
         type Address = sp_runtime::MultiAddress<AccountId, ()>;
 
-        type Migrations = ();
+        type TrieId = BoundedVec<u8, ConstU32<128>>;
+        type CodeHash = <Runtime as frame_system::Config>::Hash;
+
+        // `ContractInfo` struct that we need for `ClearContractsChildTries` but pallet-contracts
+        // doesn't expose publicly.
+        #[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+        pub struct ContractInfo {
+            pub trie_id: TrieId,
+            pub code_hash: CodeHash,
+            pub storage_bytes: u32,
+            pub storage_items: u32,
+            pub storage_byte_deposit: Balance,
+            pub storage_item_deposit: Balance,
+            pub storage_base_deposit: Balance,
+            pub delegate_dependencies:
+                BoundedBTreeMap<CodeHash, Balance, ContractsMaxDelegateDependencies>,
+        }
```

Comment:
Verifiable [here](https://github.com/paritytech/polkadot-sdk/blob/65a434a0ed474c14f692dcf9f69f8da66a99d401/substrate/frame/contracts/src/storage.rs#L45-L73) and [here](https://github.com/paritytech/polkadot-sdk/blob/65a434a0ed474c14f692dcf9f69f8da66a99d401/substrate/frame/contracts/src/lib.rs#L156-L157). I see the visibility level only of `trie_id`, `code_hash`, `storage_byte_deposit` to be public, but not the other fields.